### ISSUE TITLE
Was just deceived by the wallet

### ISSUE THREAD

**Issue by pocesar** [clemahieu/raiblocks#514]
I've withdrawn 29.45047 from Kucoin https://raiblocks.net/block/index.php?h=FB9E672D51E41C5D70B1F550BD8BD11711D50A0526E18574023E1B7B26293B93 to the wrong address because Ctrl + C failed to copy the account on raiwallet, it instead kept the clipboard of an address of an old transaction from the main window from raiwallet that happened before.... WHAT NOW?
it was supposed to have copied xrb_3fi16pk4gyz331r4531m7jywrfsgp3h31yoyfusac77esuamh65r5kwjz7dm

**Comment by zaycc** [comment]
I know it may not be what you want to hear....but this has nothing to do with raiwallet or raiblocks in general, this is a user error. The best option you may do is try to locate the owner of that wallet to see if they will be generous enough to send it to the correct address. Sorry this happened to you

**Comment by androm3da** [comment]
> kept the clipboard of an address of an old transaction from the main window from raiwallet that happened before.... 

Is this other address one that you control?

**Comment by pocesar** [comment]
no.... the address it copied is from http://raiminer.com/ payment wallet (xrb_3jybgajxebuj9kby3xusmn4sqiomzu15trmkwb1xyrynnc7axss3qp1yn679), I contacted them, I'm really hoping that they can return me

**Comment by cryptocode** [comment]
@pocesar exactly how did you copy in the wallet (what part of the UI), and can you provide reproducible steps?

**Comment by pocesar** [comment]
They returned me everything! They (people at raiminer.com) were really fast to respond and then sent everything back! Completely trustworthy https://raiblocks.net/block/index.php?h=5BEBC82769ED5D45984F5EAFFA9290F87C898A1EE2A0BEE0B39AD267531F7CCF

But this should serve as a warning. In Ethereum Wallet, when you click ANYTHING that is supposed to copy something, a "Copied to the clipboard" message pops up! That feedback from the wallet (either using Windows / OSX notification center for easy of implementation) is a must. 

@cryptocode I click the Accounts > selected the account row and column and pressed Ctrl + C. I'm guessing the UI failed to receive the right message for the clipbboard copy due to high CPU (i'm nearing 98% right now) but not because rai_wallet is using a lot of CPU, I'm running X15 CPU miner with equihash GPU miner at the same time. somewhere in between, something happened. like I said, when receiving the Ctrl + C command, and it actually copied the address, give a feedback like Ethereum, but that taught me a lesson to always double check, regardless.

**Comment by cryptocode** [comment]
@pocesar Works for me (on macOS). But some visual feedback that the copy was performed is a good idea anyway.

**Comment by brightmonkey** [comment]
Glad it worked out for you! Can you please close this issue?

**Comment by pocesar** [comment]
this issue is partially done, the "Copy" button on the main window shows "Copied!" after being pressed, but there's no feedback when using Ctrl + C on the `account` column in the ListView (which was the problem from the start)

**Comment by rotilho** [comment]
I had exactly the same problem and I luckily manage to recovery my funds thanks to NanoMiner.

I thought I copied from account list using Ctrl + C and didn't check the address before send (yep, I was stupid)
